.npmrc

```diff
@@ -1 +1,2 @@
-@rationally-app:registry=https://npm.pkg.github.com
\ No newline at end of file
+@rationally-app:registry=https://npm.pkg.github.com
+legacy-peer-deps=true
```

Comment:
https://github.com/npm/cli/issues/4998 npm had a previously not working breaking change that is now working, causing dependency check to be stricter than it used to be. However, most 3rd party packages havent updated to work with the stricter check

Reply:
will we revisit this next time our other packages are updated? maybe need another card for that

Reply:
errr when sdk 46/47 maybe? hahaha